tests/mountingOptions/attachTo.spec.ts

```diff
@@ -39,31 +39,33 @@ describe('options.attachTo', () => {
     const root = document.getElementById('root')
     const rendered = document.getElementById('attach-to')
     expect(wrapper.vm.$el.parentNode).not.toBeNull()
-    expect(root).toBeNull()
+    expect(root).not.toBeNull()
     expect(rendered).not.toBeNull()
     expect(rendered.outerHTML).toBe(outerHTML)
     wrapper.unmount()
     expect(document.getElementById('attach-to')).toBeNull()
   })
 
-  it('correctly hydrates markup', () => {
-    expect(document.getElementById('attach-to')).toBeNull()
+  it.todo('correctly hydrates markup')
 
-    const div = document.createElement('div')
-    div.id = 'attach-to'
-    div.setAttribute('data-server-rendered', 'true')
-    div.innerHTML = innerHTML
-    document.body.appendChild(div)
-    expect(div.outerHTML).toBe(ssrHTML)
-    const wrapper = mount(TestComponent, {
-      attachTo: '#attach-to'
-    })
+  // it('correctly hydrates markup', () => {
+  //   expect(document.getElementById('attach-to')).toBeNull()
 
-    const rendered = document.getElementById('attach-to')
-    expect(wrapper.vm.$el.parentNode).not.toBeNull()
-    expect(rendered).not.toBeNull()
-    expect(rendered.outerHTML).toBe(outerHTML)
-    wrapper.unmount()
-    expect(document.getElementById('attach-to')).toBeNull()
-  })
+  //   const div = document.createElement('div')
+  //   div.id = 'attach-to'
+  //   div.setAttribute('data-server-rendered', 'true')
```

Comment:
Should we just remove this entirely (not a huge fan of keeping old code around unless we know we will use it for sure).